sockets/delegateMonitor.js

```diff
@@ -237,27 +242,31 @@ module.exports = function (app, connectionHandler, socket) {
 								existing.blocks = res.blocks;
 								existing.blocksAt = moment();
 								existing = updateDelegate(existing, false);
+
+								running.getLastBlocks = false;
```

Comment:
Why set this here?

Reply:
`running.getLastBlocks` keeps being set to `true` for some reason, so the `data.lastBlock.block.height` gets never updated. I'm going to investigate it again.